app/environment.js

```diff
@@ -3,12 +3,14 @@ const environment = Object.assign({
   DEVELOPMENT: 'development',
   TEST: 'test',
   PRODUCTION: 'production',
+  NETWORK: process.env.NETWORK || 'development', // options: mainnet | testnet | staging | development
```

Comment:
Defaults for environment params are all set in the [base webpack config](https://github.com/input-output-hk/daedalus/blob/master/webpack/webpack.config.base.js#L44) atm. I don't fully remember the reason but let's stick to that for consistency.